aws/data_source_aws_route53_resolver_rule_test.go

```diff
@@ -4,11 +4,16 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/aws/aws-sdk-go/service/route53resolver"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+func init() {
+	RegisterServiceErrorCheckFunc(route53resolver.EndpointsID, testAccErrorCheckSkipRoute53)
+}
```

Comment:
Certainly not for this set of pull requests but my hunch is that we will want to have a standard internal/service/X/ (e.g. `internal/service/route53resolver/errorcheck.go`) place to add these rather than "random" acceptance testing files. 👍

Reply:
Yes! I like it.